bin/batch.sh

```diff
@@ -3,4 +3,4 @@ conf_file=${1:-"conf/batch.yaml"}
 
 set -x
 
-java ${JAVA_OPTS} -cp "src/main/resources/:target/classes:target/lib/*:target/dependency/*" macrobase.MacroBase batch $conf_file
+java ${JAVA_OPTS} -cp "src/main/resources/:target/classes:target/lib/*:target/dependency/*" macrobase.MacroBase pipeline $conf_file
```

Comment:
should we just merge `bin/batch.sh` and `bin/streaming.sh`? Or we could force them to be batch and streaming using command line arguments, currently they are pretty much identical.